src/popup/components/InputField.vue

```diff
@@ -143,6 +150,10 @@ export default {
       }
       return !!this.message;
     },
+    availableTextLimit() {
+      return (this.textLimit && this.value?.length)
+        ? this.textLimit - this.value.length : this.textLimit;
```

Comment:
Maybe we can agree on one approach of writing one line conditionals?
For me personally it's way better to follow with eyes the conditions written in following form:
```
return (someTest)
  ? firstResult
  : secondResult;
```
It's like a poll: question on the first line and the possible answers on separate lines.